frappe/public/js/frappe/views/communication.js

```diff
@@ -206,6 +206,11 @@ frappe.views.CommunicationComposer = Class.extend({
 		this.dialog.fields_dict.subject.set_value(this.subject || '');
 
 		this.setup_earlier_reply();
+
+		if ('frm' in this && !this.is_a_reply) {
+			// set default email template for the first email in a document
+			this.dialog.set_value("email_template", this.frm.meta.default_email_template || '');
+		}
```

Comment:
@barredterra 
The issue @rtdany10 mentioned is evidently occurring on re-triggering the email template. 
Although this should be fixed on the email template trigger level, this snippet is only making the issue more prominent.
Not very many people manually reset the template all the time.

Can we temporarily handle it here by checking if `get_last_edited_communication()` exists ? If yes, that means this dialog has been re-opened from its draft state and hence do not re-trigger email template since it has already been triggered before and populated.

Or we can set the current email template in a `frappe.last_set_email_template` and trigger the email template if the value differs, something along those lines.

Reply:
@barredterra Please handle the comment above too

Reply:
Not sure how this can be solved. I'd need some help here.